src/DssDeploy.sol

```diff
@@ -365,6 +362,17 @@ contract DssDeploy is DSAuth {
         end.rely(address(pause.proxy()));
     }
 
+    function deployESM(address gov, uint256 min) public auth {
+        require(address(pause) != address(0), "Missing previous step");
```

Comment:
We should probably check if we have the `end` set here too.

Reply:
Not really as you are checking the `pause`, which to be deployed needs the `end`. It is a chain of the restrictions.